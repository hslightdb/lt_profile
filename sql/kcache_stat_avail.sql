SELECT count(1) > 0 FROM profile.sample_kcache;
SELECT count(1) > 0 FROM profile.sample_kcache_total;
