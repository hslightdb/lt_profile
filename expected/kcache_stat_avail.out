SELECT count(1) > 0 FROM profile.sample_kcache;
 ?column? 
----------
 t
(1 row)

SELECT count(1) > 0 FROM profile.sample_kcache_total;
 ?column? 
----------
 t
(1 row)

